assets/scss/components/_header.scss

```diff
@@ -26,3 +23,36 @@
 .c-header > nav .has-dropdown.is-open + ul {
     display: block;
 }
+
+.c-header-announcement-banner {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-black;
+    padding: 16px 24px;
+
+    p {
+        color: $color-white;
+        text-align: center;
+        font-family: $font-regular;
+        font-size: 16px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 1.5;
+        margin: 0;
+    }
+
+    strong {
+        font-weight: 600;
+    }
+
+    a {
```

Comment:
Similar here, if css class can be applied, avoid nesting selectors and styling directly an element